app.js

```diff
@@ -223,8 +227,8 @@ function updateUser(user) {
       }
     }
   );
-}
-function upsertUser(req, res) {
+};
+var upsertUser = function(req, res) {
```

Comment:
could you use `const` instead of `var`? Ideally we should prefer `const` and `let` to `var` which has a more confusing behaviour (global scoping)

Reply:
Yeah I understand.
I'll make the changes ASAP.